modules/backup-and-restore/pages/point-in-time-restore.adoc

```diff
@@ -0,0 +1,80 @@
+- Example: If the first full backup was made on 2024-06-01T00:00:00Z, you cannot restore to any time before this, such as 2024-05-31T23:59:59Z. Similarly, if the current time is 2024-06-15T12:00:00Z, you cannot restore to a time like 2024-06-16T00:00:00Z.
+* Time Points Not Covered by Existing Backups
+- The selected time point must be within the range of an existing backup’s coverage. A time point is considered "not covered" if it does not fall within the time range defined by any backup's POINT-IN-TIME START TIME and CREATED AT timestamps. You can verify the backup’s time ranges by running the gadmin backup list command. Here is an example:
+
+[console, gsql]
+---
+$ gadmin backup list
+|================================================================================================================================================
+|             TAG              |    TYPE     | VERSION |  SIZE  |             BASE             | POINT-IN-TIME START TIME |      CREATED AT      
+| backup-2024-06-13T061939.131 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T06:19:39Z     | 2024-06-13T06:19:39Z 
+| backup-2024-06-13T063915.054 | INCREMENTAL | 4.1.0   | 1.5 MB | backup-2024-06-13T061939.131 | 2024-06-13T06:19:39Z     | 2024-06-13T06:39:15Z 
+| backup-2024-06-13T081939.120 | FULL        | 4.1.0   | 1.5 MB |                              | 2024-06-13T08:19:39Z     | 2024-06-13T08:19:39Z 
+|================================================================================================================================================
+---
+
+In this example, the time range for backup `backup-2024-06-13T061939.131` is `[2024-06-13T06:19:39Z,2024-06-13T06:19:39Z]`, therefore, for a full backup is just a point. As for the incremental backup `backup-2024-06-13T063915.054`, its range is `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]`. `2024-06-13T06:35:00Z` is supported for it is covered by the time range `[2024-06-13T06:19:39Z , 2024-06-13T06:39:15Z]` of the backup `backup-2024-06-13T063915.054`. While, users can’t restore to the timepoint `2024-06-13T07:00:00Z` directly, for it is not in any backup's time range. However, if no restore was performed since `2024-06-13T07:00:00Z`, users may be able to create a new differential backup at right now and then restore to `2024-06-13T07:00:00Z`. 
+
+
+[NOTE]
+====
+because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  we’d better ensure the possible time points are all covered by existing backups. if there is not this kind of backup, you may create a by `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
```

Comment:
```suggestion
because the restore action will make the uncovered timepoint unavailable to restore, if you are not very sure about the time point to restore,  it is best to ensure the possible time points are all covered by existing backups. If you do not yet have an appropriate backup, you may create one by running `gadmin backup create <backup_tag> --incremental --base <full_backup_tag>`, the new backup will cover the range from the full backup <full_backup_tag>'s create time to the current time.
```